packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -66,21 +66,20 @@ const GlobalClassNames = {
 // TODO: Audit perf. impact of and potentially remove memoizeFunction.
 //       https://github.com/OfficeDev/office-ui-fabric-react/issues/5534
 export const getItemClassNames = memoizeFunction(
```

Comment:
the original code had memoize too?

I would add jsdoc stating that this is package-internal method that has unfortunately been depended upon and is being kept in this form for backwards compat, but will be cleaned up by Fabric 7. 

Also add `@deprecated`.

Reply:
Yea, it used `memoizeFunction` prior to #6094. Regarding the annotation, I'd prefer to use `@internal` in preparation for API Extractor. This function does all of the `getStyles` style-set merging so its not `@deprecated` per say. However it should not be relied on as a _public_ API. It is our internal mechanism for generating `ContextualMenuItem` classNames given style props.